pkg/pipelines/pipelines.go

```diff
@@ -398,3 +400,56 @@ func GetLatestPipelinerun(c *clients.Clients, namespace string) (string, error)
 	return prs.Items[0].Name, nil
 
 }
+
+func CheckLogVersion(c *clients.Clients, binary, namespace string) {
+	prname, err := GetLatestPipelinerun(store.Clients(), store.Namespace())
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun: %v", err))
+		return
+	}
+	// Get PipelineRun logs
+	logsBuffer, err := getPipelinerunLogs(c, prname, namespace)
+	if err != nil {
+		testsuit.T.Fail(fmt.Errorf("Failed to get PipelineRun logs: %v", err))
+		return
+	}
+	logs := logsBuffer.String()
+
+	switch binary {
+	case "tkn-pac":
+		expectedVersion := os.Getenv("PAC_VERSION")
+
+		// Convert logs to string and extract tkn-pac version using regex
+		re := regexp.MustCompile(`\b(\d+\.\d+\.\d+)\b`)
+		logVersion := re.FindAllString(logs, -1)
+		parts := strings.Split(logVersion[0], ".")
+		if len(parts) < 2 {
+			testsuit.T.Fail(fmt.Errorf("Invalid tkn-pac Version: %s", logVersion[0]))
+			return
+		}
+		logVersion[0] = strings.Join(parts[:2], ".")
+
+		// Compare log version with expected version
+		if logVersion[0] != expectedVersion {
+			testsuit.T.Fail(fmt.Errorf("tkn-pac has an unexpected version: %s, Expected %s", logVersion[0], expectedVersion))
+		}
+
+	case "tkn":
+		expectedVersion := os.Getenv("OSP_VERSION")
```

Comment:
There is a bug in task in 1.18.0, it should print upstream CLI version, see https://issues.redhat.com/browse/SRVKP-7457

Please switch this line to use the correct variable.

Reply:
Done